platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -614,13 +629,19 @@ fn build_solana_package(
 
         match cdylib_targets.len() {
             0 => {
-                warn!(
-                    "Note: {} crate does not contain a cdylib target",
+                info!(
+                    "Note: {} crate is a library to be imported by other solana programs",
```

Comment:
nit: I think the previous message was clearer here, since using `cargo build-sbf` on a bin package will now produce an incorrect message.

Maybe we can clarify the previous message, ie: "Note: {} crate does not contain a cdylib target, so it cannot be built as a Solana program."

Reply:
> Maybe we can clarify the previous message, ie: "Note: {} crate does not contain a cdylib target, so it cannot be built as a Solana program."

That message sounds alarming as if the package won't be built because it is not a cdylib. The point is only to inform that this is not going to be a Solana program, but instead a library. Perhaps I should remove it altogether?

Reply:
Removed in https://github.com/anza-xyz/agave/pull/5729/commits/24284ddacc1bd1fee9a4df337d8ae030f5f9635e